roles/testnode/tasks/yum/packages.yml

```diff
@@ -57,6 +57,13 @@
     enablerepo: epel
   when: epel_packages|length > 0
 
+- name: Install copr packages
```

Comment:
why is this mentioning "copr"?  if it's epel?

Reply:
Ah, sorry, I forgot to remove this.